src/notation/view/notationviewinputcontroller.cpp

```diff
@@ -517,6 +607,8 @@ void NotationViewInputController::mouseDoubleClickEvent(QMouseEvent* event)
 
     if (element->isTextBase()) {
         viewInteraction()->startEditText(element, m_view->toLogical(event->pos()));
+    } else {
+        viewInteraction()->startEditElement();
```

Comment:
whether or not we need to support "element edit mode" is still under discussion - but for now, it is in order to enable using the keyboard to make offset/leading space adjustments